src/main/resources/assets/openblocks/lang/de_de.lang

```diff
@@ -303,10 +303,10 @@ tile.openblocks.sky.normal.description=Bringe das Außen nach innen mit dem neue
 tile.openblocks.sky.inverted.name=Invertierter Himmelsblock
 
 tile.openblocks.radio.name=Radio
-tile.openblocks.radio.description=Das Radio erlaubt es dir, Internetradio zu spielen! Zunächst brauchst du Wellenkristalle, die du in Dungeon-Truhen findest oder von einem bestimmten Dorfbewohner bekommst. Platziere den Kristall ins Radio und gib diesem ein Redstone-Signal, um es anzuschalten.  Die Stärke des Redstone-Signals bestimmt die Lautstärke.\n\nIn der Konfigurations-Datei des Servers kannst du konfigurieren, welche Radiosender verfügbar sind.
+tile.openblocks.radio.description=Das Radio erlaubt es dir, Internetradio zu spielen! Zunächst brauchst du Wellenkristalle, die du in Truhen aus Verliesen findest oder von einem bestimmten Dorfbewohner bekommst. Platziere den Kristall ins Radio und gib diesem ein Redstone-Signal, um es anzuschalten. Die Stärke des Redstone-Signals bestimmt die Lautstärke.\n\nIn der Konfigurations-Datei des Servers kannst du konfigurieren, welche Radiosender verfügbar sind.
 
 tile.openblocks.xp_shower.name=XP-Dusche
-tile.openblocks.xp_shower.description=Bringe eine XP-Dusche an einen Tank voller XP-Saft an und sie wird XP-Orbs ausschütten. Diese Orbs können von einem XP-Gitter oder einer MFR-Kloake aufgesammelt werden.
+tile.openblocks.xp_shower.description=Bringe eine XP-Dusche an einen Tank voller Erfahrungs-Saft an und sie wird Erfahrungs-Kugeln ausschütten. Diese Kugeln können von einem Erfahrungspunkte-Gitter oder einer MFR-Kloake aufgesammelt werden.
```

Comment:
I really dislike that you translated XP to... this.

Reply:
I change it, but what is about "Erfahrungs-Kugeln"?

Reply:
Minecraft's own translation for it is "Erfahrungskugel". I'd just remove the hyphen, same with the "Saft".

Reply:
Ok then so.